frontend/src/metabase/dashboard/components/ClickBehaviorSidebar/LinkOptions/LinkedEntityPicker/LinkedEntityPicker.tsx

```diff
@@ -268,15 +266,15 @@ export function LinkedEntityPicker({
               title={getModalTitle()}
               onClose={hasSelectedTarget ? onClose : undefined}
             >
-              {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
-              {/* @ts-ignore */}
+              {/* TODO: drop maxHeight when PickerComponent is migrated to TS */}
               <PickerComponent
                 filterPersonalCollections={filterPersonalCollections}
                 value={clickBehavior.targetId}
                 onChange={(targetId: CardId | DashboardId) => {
                   handleSelectLinkTargetEntityId(targetId);
                   onClose();
                 }}
+                maxHeight={undefined}
```

Comment:
This error is given incorrectly (due to `@types/prop-types` package) and can be fixed by refactoring `QuestionPicker` to TS.